### PR TITLE
fix: add fee zero adjustments to sign/place order examples

### PR DESCRIPTION
**Pull request by alfetopito**
# Description

Users are confused due to fee-zero changes.

The sign/order placement tutorial no longer works, as the feeAmount must be set to `0`.

# Changes

- [ ] Update signing order tutorial https://learn-git-fix-fee-zero-cowswap.vercel.app/tutorial/sign-order
- [ ] Update placing order tutorial https://learn-git-fix-fee-zero-cowswap.vercel.app/tutorial/submit-order

# Testing

Make sure shared test steps work.

### CODE REVIEW

**Comment by alfetopito** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by alfetopito** [comment]
recheck

File: content/tutorial/01-simple-orders/01-order/03-sign-order/README.md [hunk 1]
```diff
@@ -20,12 +20,18 @@ For signing, we will use the `UnsignedOrder` type from the `SDK`, along with the
 /// file: run.ts
 import type { Web3Provider } from '@ethersproject/providers';
 +++import { OrderBookApi, SupportedChainId, OrderQuoteRequest, OrderQuoteSideKindSell, OrderSigningUtils, UnsignedOrder } from '@cowprotocol/cow-sdk';+++
++++import { BigNumber } from 'ethers';+++
```
Comment (anxolin) [inline content/tutorial/01-simple-orders/01-order/03-sign-order/README.md:23]: what are all these ++++?

File: content/tutorial/01-simple-orders/01-order/03-sign-order/README.md [hunk 2]
```diff
@@ -20,12 +20,18 @@ For signing, we will use the `UnsignedOrder` type from the `SDK`, along with the
...
+
+    
 
 export async function run(provider: Web3Provider): Promise<unknown> {
   // ...
 
   const order: UnsignedOrder = {
     ...quote,
+    sellAmount: BigNumber.from(quote.sellAmount).add(BigNumber.from(quote.feeAmount)).toString(), // Quoted sellAmount must be added to quoted feeAmount
+    feeAmount: '0', // and feeAmount must be set to 0
```
Comment (anxolin) [inline content/tutorial/01-simple-orders/01-order/03-sign-order/README.md:34]: Maybe mention that the the fee is already priced in the buy amount.

Also, I would make a const `buyAmount` for readability

File: content/tutorial/01-simple-orders/01-order/03-sign-order/app-b/src/lib/run.ts [hunk 3]
```diff
@@ -27,8 +28,12 @@ export async function run(provider: Web3Provider): Promise<unknown> {
 
     const { quote } = await orderBookApi.getQuote(quoteRequest);
 
+    // Quoted sellAmount must be added to quoted feeAmount, and feeAmount must be set to 0
+
     const order: UnsignedOrder = {
       ...quote,
+      sellAmount: BigNumber.from(quote.sellAmount).add(BigNumber.from(quote.feeAmount)).toString(),
+      feeAmount: '0',
```
Comment (anxolin) [inline content/tutorial/01-simple-orders/01-order/03-sign-order/app-b/src/lib/run.ts:36]: same as above

File: content/tutorial/01-simple-orders/01-order/03-sign-order/README.md (hunk 1 above)
Comment (alfetopito) [reply content/tutorial/01-simple-orders/01-order/03-sign-order/README.md:23]: Piggybacking on the existing code, it adds a highlight to the row:

image

Comment (alfetopito) [reply content/tutorial/01-simple-orders/01-order/03-sign-order/README.md]: Anyway, no longer using `BigNumber`, removed the import.

**Pull request merged**